tools/integration_tests/write_large_files/write_large_files_test.go

```diff
@@ -46,9 +48,38 @@ func compareFileFromGCSBucketAndMntDir(gcsFile, mntDirFile, localFilePathToDownl
 	diff, err := operations.DiffFiles(mntDirFile, localFilePathToDownloadGcsFile)
 	if diff != 0 {
 		err = fmt.Errorf("Download of GCS object %s didn't match the Mounted local file (%s): %v", localFilePathToDownloadGcsFile, mntDirFile, err)
-		return
+		return err
 	}
-	return
+	return err
+}
+
+// Write data of chunkSize in file at given offset.
+func WriteChunkSizeInFile(file *os.File, chunkSize int, offset int64) (err error) {
```

Comment:
Does this need to be a public  function? I don't see to expose this function from outside this file. Either please make this private or move this to file_operations.go .

Reply:
made it private.